### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by maksimpro001**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    pep = [Person(p["name"], p["age"]) for p in people]
+    for i, pop in enumerate(people):
```
Comment (ArsenPidhoretskyi) [inline app/main.py:12]: pop is not good variable name

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    pep = [Person(p["name"], p["age"]) for p in people]
```
Comment (ArsenPidhoretskyi) [inline app/main.py:11]: p, pep - bad variable names

File: app/main.py [hunk 3]
```diff
@@ -1,8 +1,20 @@
...
+    for i, pop in enumerate(people):
+        try:
+            if pop["wife"] is not None:
+                pep[i].wife = Person.people[pop["wife"]]
+        except KeyError:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:16]: why do you need to use try except? It is redundant, use correct if statements

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Review by YukihiroSM — APPROVED** [review]
good job!

**Pull request closed without merging**